examples/cinematography_domain/usage.py

```diff
@@ -30,19 +31,45 @@ def _upload_data(client: CineClient) -> None:
         ),
         Movie(
             externalId="movie2",
-            title="Star Wars: Episode IV – A New Hope",
-            release=Timestamp("1977-05-25T00:00:00-06"),
-            director=Person(externalId="person4", name="George Lucas"),
+            title="Thor",
+            release=Timestamp("2011-04-17T00:00:00+10"),
+            director=Person(externalId="person4", name="Kenneth Branagh"),
             actors=[
-                Person(externalId="person5", name="Mark Hamill"),
-                Person(externalId="person6", name="Harrison Ford"),
-                Person(externalId="person7", name="Carrie Fisher"),
+                Person(externalId="person5", name="Chris Hemsworth"),
+                Person(externalId="person6", name="Natalie Portman"),
+                Person(externalId="person7", name="Tom Hiddleston"),
             ],
-            meta=JSONObject({"run_time": 121}),
+            meta=JSONObject({"run_time": 114}),
         ),
     ]
     client.movie.apply(movies)
 
+    # add another movie, showing the use of _reference
+    ragnarok = Movie(
+        externalId="movie3",
+        title="Thor: Ragnarok",
+        release=Timestamp("2017-10-10T00:00:00-08"),
+        director=Person(externalId="person8", name="Taika Waititi"),
+        actors=[
+            # These persons already exist. We need not fill all fields
+            # only externalId and set _reference=True.
+            # Note: we could instead fill all fields instead (just "name" in
+            # this example), and the API would update the fields. By using
+            # _reference=True, we don't update those field, only create the relations
+            Person(externalId="person5", _reference=True),
+            Person(externalId="person7", _reference=True),
```

Comment:
Why do you have reference with a underscore prefix? What are you trying to signa?

Reply:
To avoid polluting the namespace of attributes that the user can set up on their models.

Reply:
`mypy` ganged up with `pydantic.mypy`, they hurt my feelings and forced me to find a better API 😑  
`Person.ref(externalId="...")` instead of `Person(externalId="...", _reference=True)`.